config/locales/en/helpers.yml

```diff
@@ -72,10 +73,14 @@ en:
             more information to complete your claim. If you add alternative contact
             details we’ll use those instead.
       steps_case_details_form:
+        remitted_to_magistrate_options:
+          'yes': We’ll ask you to upload evidence of this before you submit this claim
         assigned_counsel_options:
           'yes': We'll ask you to upload your CRM8 form before you submit this claim
+        assigned_counsel: We'll ask you to upload your CRM8 form before you submit this claim
```

Comment:
seems strange this note is here for `assigned_counsel_options` and `assigned_counsel`

can we remove the one that isn't needed

Reply:
removed